src/mjolnir/landmark_builder.cc

```diff
@@ -1,10 +1,63 @@
-#include "mjolnir/landmark_database_builder.h"
+#include "mjolnir/landmark_builder.h"
 #include "filesystem.h"
+#include "mjolnir/osmpbfparser.h"
 #include "mjolnir/util.h"
+#include <boost/property_tree/ptree.hpp>
+
+namespace {
+struct landmark_callback : public OSMPBF::Callback {
+public:
+  landmark_callback(const std::string& db_name) : db_(db_name, false) {
+  }
+  virtual ~landmark_callback() {
+  }
+
+  virtual void
+  node_callback(const uint64_t /*osmid*/, double lng, double lat, const OSMPBF::Tags& tags) override {
+    valhalla::mjolnir::Landmark landmark;
```

Comment:
this wasn't addressed yet: we don't need this intermediate struct at all. just put the properties into the `insert_landmark` call.

Reply:
let's keep it unresolved until we can see it's done :) best to let the reviewers resolve comments, then they don't lose the overview of their own comments

Reply:
no problem. thanks for the tip! 
and it's fixed now.